downloaded/commit_hash.sh

```diff
@@ -1 +1,3 @@
 COMMIT_HASH=7252b1aeb598c51308d44a0e3ca48aa84b036b59
+COMMIT_HASH=${COMMIT_HASH}
```

Comment:
I'm afraid I don't really understand this change. is it intentional?

Reply:
asking aserto-bot...